### ISSUE TITLE
Incorrectly detecting kernels

### ISSUE THREAD

**Issue by dopplershift** [Anaconda-Platform/nb_conda_kernels#64]
On my system, my root conda environment has nbformat installed, but no ipykernel (to run nbdime). When I open a new notebook with my jupyter notebook environment and nb_conda_kernels, it insists on trying to start a kernel from my root environment, failing with:
```
/Users/rmay/miniconda3/bin/python: No module named ipykernel
[W 15:04:35.832 NotebookApp] 404 GET /nbextensions/widgets/notebook/js/extension.js?v=20170309150426 (::1) 6.21ms referer=http://localhost:8888/notebooks/notebook-sandbox/Intro_to_Gridded_Plotting.ipynb
[I 15:04:38.731 NotebookApp] KernelRestarter: restarting kernel (1/5)
/Users/rmay/miniconda3/bin/python: No module named ipykernel
[I 15:04:41.738 NotebookApp] KernelRestarter: restarting kernel (2/5)
/Users/rmay/miniconda3/bin/python: No module named ipykernel
[I 15:04:44.745 NotebookApp] KernelRestarter: restarting kernel (3/5)
/Users/rmay/miniconda3/bin/python: No module named ipykernel
[W 15:04:45.802 NotebookApp] Timeout waiting for kernel_info reply from 841335c4-39d4-4976-b242-3f39d96dedc8
[I 15:04:47.753 NotebookApp] KernelRestarter: restarting kernel (4/5)
WARNING:root:kernel 841335c4-39d4-4976-b242-3f39d96dedc8 restarted
/Users/rmay/miniconda3/bin/python: No module named ipykernel
[W 15:04:50.764 NotebookApp] KernelRestarter: restart failed
```

The code incorrectly assumes that the presence of `bin/jupyter` and `bin/python` implies that ipykernel is available. (nb_conda_kernels/manager.py:115)

**Comment by damianavila** [comment]
Thanks for the report... we should handle this better to filter our these kind of cases.

**Comment by mcg1969** [comment]
I'm working on this in #80

**Comment by mcg1969** [comment]
Fixed in #80
